.travis.yml

```diff
@@ -0,0 +1,31 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+  - windows
+language: rust
+rust:
+  - 1.29.2
+sudo: false
+cache:
+  cargo: true
+before_script:
+  - if ! cargo prune --version | grep -q "0.1.8"; then
+      cargo install cargo-prune --vers="0.1.8" --force;
+    fi
+  - if ! rustfmt --version | grep -q "0.99.1-stable"; then
+      rustup component add rustfmt-preview;
+    fi
+  - if ! cargo clippy --version | grep -q "0.0.212"; then
```

Comment:
This clippy version works with 1.29.0. Does it also work with 1.29.2 ? Just trying to make sure.

Reply:
It should, that's my setup:
```
➜  ~ rustc --version
rustc 1.29.2 (17a9dc751 2018-10-05)
➜  ~ rustfmt --version
rustfmt 0.99.1-stable (da17b68 2018-08-04)
➜  ~ cargo clippy --version
0.0.212
```